alluxio-ui/master/src/containers/pages/Overview/Overview.tsx

```diff
@@ -0,0 +1,220 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+import {AxiosResponse} from 'axios';
+import React from 'react';
+import {connect} from 'react-redux';
+import {Alert, Progress, Table} from 'reactstrap';
+import {Dispatch} from 'redux';
+
+import {IScopedPropertyInfo, IStorageTierInfo} from '../../../constants';
+import {IApplicationState} from '../../../store';
+import {fetchRequest} from '../../../store/overview/actions';
+import {IOverview} from '../../../store/overview/types';
+
+interface IPropsFromState {
+  errors?: AxiosResponse;
+  loading: boolean;
+  overview: IOverview;
+  refresh: boolean;
+}
+
+interface IPropsFromDispatch {
+  fetchRequest: typeof fetchRequest;
+}
+
+type AllProps = IPropsFromState & IPropsFromDispatch;
+
+class Overview extends React.Component<AllProps> {
+  public componentWillUpdate(prevProps: AllProps) {
+    if (this.props.refresh !== prevProps.refresh) {
+      this.props.fetchRequest();
+    }
+  }
+
+  public componentWillMount() {
+    this.props.fetchRequest();
+  }
+
+  public render() {
+    const {errors, overview} = this.props;
+
+    if (errors) {
```

Comment:
If we are debugging this, where can we find the actual error?

Reply:
Errors will have the actual response object from the api. It will be within this object.